cnd/src/config/settings.rs

```diff
@@ -410,4 +521,48 @@ mod tests {
 
         assert_that(&settings).is_err();
     }
+
+    #[test]
+    fn given_network_on_cli_when_config_disagrees_then_error() {
```

Comment:
Thank you for finally adding tests for this <3

Reply:
I just moved the tests :)